NuGet.config

```diff
@@ -4,7 +4,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="myget.org roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
+    <add key="myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
I think this is nightly? I can't find the original repo (roslyn-master-nightly) on myget, and the roslyn repo appears to be updated with every build. Am I missing permissions?